examples/azure/terraform-azure-sqldb-example/variables.tf

```diff
@@ -0,0 +1,43 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# ENVIRONMENT VARIABLES
+# Define these secrets as environment variables
+# ---------------------------------------------------------------------------------------------------------------------
+
+# ARM_CLIENT_ID
+# ARM_CLIENT_SECRET
+# ARM_SUBSCRIPTION_ID
+# ARM_TENANT_ID
+
+# ---------------------------------------------------------------------------------------------------------------------
+# REQUIRED PARAMETERS
+# You must provide a value for each of these parameters.
+# ---------------------------------------------------------------------------------------------------------------------
+
+# ---------------------------------------------------------------------------------------------------------------------
+# OPTIONAL PARAMETERS
+# These parameters have reasonable defaults.
+# ---------------------------------------------------------------------------------------------------------------------
+
+variable location {
+  description = "The supported azure location where the resource exists"
+  type        = string
+  default     = "West US2"
+}
+
+variable sqlserver_admin_login {
+    description = "The administrator login name for the sql server."
+    type        = string
+    default     = "3cx1n1z5r079b"
```

Comment:
Administrator is not a valid login name in this version of SQL Server, using it will cause the create sql server API to return 400-InvalidLoginName error https://docs.microsoft.com/en-us/rest/api/sql/managedinstances/update

Reply:
ah, we should just make it something a little more meaningful.

Reply:
fixed